recipe/build.sh

```diff
@@ -102,9 +102,11 @@ cmake .. -LAH                                                             \
     -DWITH_FFMPEG=1                                                       \
     -DWITH_MATLAB=0                                                       \
     -DWITH_VTK=0                                                          \
+    -DWITH_QT=5                                                           \
     -DWITH_GPHOTO2=0                                                      \
     -DINSTALL_C_EXAMPLES=0                                                \
     -DOPENCV_EXTRA_MODULES_PATH="../opencv_contrib-$PKG_VERSION/modules"  \
+    -DBUILD_opencv_cvv=OFF                                                \
```

Comment:
Yes you can set up local CI by just following the normal instructions for setting up the CI and then setting up a MR against your own branch. But to actually make uploads you'd have to follow the scripts.

@jakirkham Is there some kind of dev branch we can upload to for local testing?